index.js

```diff
@@ -356,9 +356,12 @@ function makeVaultClient(baseUrl, fetch, token) {
         return fetchWrapper(canonicalizedBaseiUrl + path, fetch, token, options);
     }
 
-    return {
+    var api = {
+        createOtp: function() {
+            return vaultFetch('otp', { method: 'POST' });
```

Comment:
what does it return? `otp` string?

Reply:
Good question. I've adjusted the `createOtp` method to return the OTP string itself.